tests/utils.py

```diff
@@ -168,6 +168,18 @@ def assert_function_in_collapsed(function_name: str, collapsed: StackToSampleCou
     assert is_function_in_collapsed(function_name, collapsed), f"function {function_name!r} missing in collapsed data!"
 
 
+def assert_ldd_version_container(container: Container, version: str) -> None:
+    pattern = r"^ldd \(.*\) (\d*.\d*)"
+    command = "ldd --version"
+    exec_output = container.exec_run(command).output.decode("utf-8")
```

Comment:
```suggestion
    exec_output = container.exec_run("ldd --version").output.decode("utf-8")
```

No need for extra variables in this case IMO, it makes the code unnecessarily longer